### DISCUSSION TITLE
How to add a list of features into --x-feature

### DISCUSSION THREAD

**Discussion by aungthetnaing** [microsoft/vcpkg#40438] [Q&A]
In manifest mode:

How to pass multiple features. 

In CMake itself, I could pass this way:

-DVCPKG_MANIFEST_FEATURES=feature1;feature2

What's the counterpart for --x-feature. I have tried the following and both failed. 

--x-feature=feature1;feature2

--x-feature=feature1,feature2

**Comment by autoantwort** [comment]
Should be `--x-feature=feature1 --x-feature=feature2`
